### ISSUE TITLE
Question: About Finetuning SAM

### ISSUE THREAD

**Issue by Elsword016** [halleewong/ScribblePrompt#1]
Did you finetune SAM with points and bounding boxes or just with bounding boxes? Also, if only bounding boxes were used for fine-tuning, so does it work with clicks/points just out of the box?? 
Thanks

**Comment by halleewong** [comment]
Hi, we fine-tuned SAM on points, bounding boxes, **and** scribbles (represented as a series of points). So ScribblePrompt-SAM works out of the box with all three types of prompts (and combinations thereof) as well as mask inputs.

**Comment by Elsword016** [comment]
Thank you for getting back to me. Will you be releasing any training script, I would love to fine-tune the model for my images

**Comment by halleewong** [comment]
Yes, we are planning to release training code! I am actively working on cleaning up the code so it can be released.
